### DISCUSSION TITLE
iframe content does not loading in cypress.

### DISCUSSION THREAD

**Discussion by sakiburrahman** [cypress-io/cypress#14771] [Questions and Help] [answered]
iframe content is not loading while running a test using cypress

**Cypress Screen:**
![image](https://user-images.githubusercontent.com/4531044/105971366-a4c9cb00-60b4-11eb-9acf-f46700924bcd.png)

**Actual Site:**
![image](https://user-images.githubusercontent.com/4531044/105971397-ac896f80-60b4-11eb-885f-13aee315a0e2.png)
![image](https://user-images.githubusercontent.com/4531044/105971416-b1e6ba00-60b4-11eb-80e9-b63cac3e5d2f.png)

**Using the code:**

`const getIframeWindow = () => {
        return cy.get('#originalFormFrame').its('0.contentWindow').should('exist')
      }`

**Comment by sakiburrahman** [comment]
@eventualbuddha @jbr @lorennorman @kevinold  Need your help.

**Comment by Jadedy** [comment]
Have you try installing an extension in cypress chrome ?
This one works for me > https://chrome.google.com/webstore/detail/ignore-x-frame-headers/gleekbfjekiniecknbkamfmkohkpodhe

**Comment by bahmutov** [comment, marked as answer]
Working with iframes, especially cross-origin ones is tricky and requires manual workarounds. Read the blog post https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/
